packages/panels/src/Resources/Pages/Concerns/HasRelationManagers.php

```diff
@@ -55,6 +56,16 @@ public function mountHasRelationManagers(): void
         $this->activeRelationManager = array_key_first($this->getRelationManagers()) ?? null;
     }
 
+    public function renderingHasRelationManagers(View $view): void
+    {
+        $managers = $this->getRelationManagers();
+
+        if (!array_key_exists($this->activeRelationManager, $managers) && !$this->hasCombinedRelationManagerTabsWithContent()) {
+            $this->activeRelationManager = array_key_first($this->getRelationManagers()) ?? null;
+            $view->with('activeRelationManager', $this->activeRelationManager);
```

Comment:
Is this line needed? I think that all public properties are exposed to the view by default

Reply:
i'm not sure if i fully understand how the full lifecycle here works, but without setting the View-s data here, only after a following update request will the active tab set to the first item.

while trying to figure this out, i found that in the edit-record.blade.php there is a difference between $this->activeRelationManager and $activeRelationManager, (accessing the property via $this return the desired value without setting it on the viewdata) 
but can't find anything about this in the docs. i'm not sure if this is a bad pattern to rely on.

so passing :active-manager="$this->activeRelationManager to  <x-filament-panels::resources.relation-managers works, but i don't want to submit something that is not acceptable.

Reply:
I've switched to `$this->activeRelationManager`, that should be fine